src/adagrams.js

```diff
@@ -1,8 +1,149 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
-
-// Do not remove this line or your tests will break!
-export default Adagrams;
+class Adagrams {
+  static letters = {
+    "A": 9,
+    "B": 2,
+    "C": 2,
+    "D": 4,
+    "E": 12,	
+    "F": 2,
+    "G": 3,
+    "H": 2,
+    "I": 9,
+    "J": 1,
+    "K": 1,
+    "L": 4,
+    "M": 2,
+    "N": 6,
+    "O": 8,
+    "P": 2,
+    "Q": 1,
+    "R": 6,
+    "S": 4,
+    "T": 6,
+    "U": 4,
+    "V": 2,
+    "W": 2,
+    "X": 1,
+    "Y": 2,
+    "Z": 1
+  }
+
+  static letterValues = {
+    "A": 1,	
+    "B": 3,	
+    "C": 3,	
+    "D": 2,	
+    "E": 1,	
+    "F": 4,
+    "G": 2,	
+    "H": 4,	
+    "I": 1,	
+    "J": 8,	
+    "K": 5,	
+    "L": 1,	
+    "M": 3,	
+    "N": 1,
+    "O": 1,
+    "P": 3,
+    "Q": 10,
+    "R": 1,
+    "S": 1,
+    "T": 1,
+    "U": 1,
+    "V": 4,
+    "W": 4,
+    "X": 8,
+    "Y": 4,
+    "Z": 10
+  }
+
+  // Credit for shuffling function described at
+  // https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
+
+  static shuffle(array) {
+    for (let r = array.length - 1; r > 0; r--) {
+      const s = Math.floor(Math.random() * (r + 1));
+      [array[r], array[s]] = [array[s], array[r]];
+    }
+  }
+
+  // Wave 1
+  static drawLetters() {
+    let pool = [];
```

Comment:
Since you don't reassign it. 

```suggestion
    const pool = [];
```